File: daishu_seckill_interface/src/main/java/com/daishu/seckill/service/SeckillGoodsService.java
package com.daishu.seckill.service;
import com.daishu.pojo.TbSeckillGoods;

import java.util.List;

/**
 * 秒杀服务层接口
 * @author Administrator
 *
 */
public interface SeckillGoodsService {

	/**
	 * 正在参与秒杀的商品
	 * @return
	 */
	public  List<TbSeckillGoods> findList();

	/**
	 *从缓存读取
 	 */
	public  TbSeckillGoods findOneFromRedis(Long id);


	/**
	 *从缓存读取
	 */
	public  TbSeckillGoods findOne(Long id);






}

File: daishu_seckill_web/src/main/resources/static/js/service/seckillGoodsService.js
        app.service('seckillGoodsService',function ($http) {
                this.findList=function () {
                        return $http.get('seckillGoods/findList');
                }

                this.findOne=function (id) {
                    return $http.get('seckillGoods/findOneFromRedis?id='+id);
                }


             this.submitOrder=function (seckillId) {
                return $http.get('seckillOrder/submitOrder?seckillId='+seckillId);
            }
        });